packages/xo-web/src/xo-app/index.js

```diff
@@ -217,7 +217,11 @@ export default class XoApp extends Component {
 
   render() {
     const { signedUp, trial, registerNeeded } = this.props
-    const blocked = signedUp && blockXoaAccess(trial) // If we are under expired or unstable trial (signed up only)
+    // If we are under expired or unstable trial (signed up only)
+    const blocked =
+      signedUp &&
+      blockXoaAccess(trial) &&
+      !this.context.router.location.pathname.startsWith('/xoa/')
```

Comment:
It we want to let them only see the license page, we can check explicitly for ''/xoa/licenses" otherwise we can let them to manage their `XOA` (upgrade/ login with other account/ open a support tunnel ...).

Reply:
Yes, this is intended, I think it's important that the users can always access the `/xoa/*` pages.

Reply:
So the PR title and the CHANGELOG can refer that this PR allows user to manage their XOA. We can also allows the `user` page.

Reply:
Why?